litmus-portal/graphql-server/graph/schema.resolvers.go

```diff
@@ -151,8 +167,15 @@ func (r *mutationResolver) UpdateChaosWorkflow(ctx context.Context, input *model
 	return wfHandler.UpdateWorkflow(ctx, input, data_store.Store)
 }
 
-func (r *mutationResolver) DeleteClusterReg(ctx context.Context, clusterID string) (string, error) {
-	return clusterHandler.DeleteCluster(clusterID, *data_store.Store)
+func (r *mutationResolver) DeleteClusterReg(ctx context.Context, projectID string, clusterIds []*string) (string, error) {
```

Comment:
Rename to DeleteClusters. 

TODO refactor : rename Clusters to Agents in the codebase